### PR TITLE
move state changes actions to separated package; refactoring

### PR DESCRIPTION
**Pull request by msmolyakov**

### CODE REVIEW

File: src/main/java/com/wavesplatform/wavesj/Node.java [hunk 1]
```diff
@@ -540,24 +465,24 @@ public TransactionInfo getTransactionInfo(Id txId) throws IOException, NodeExcep
...
         StringEntity body = new StringEntity(JSON_MAPPER.writeValueAsString(jsonBody), StandardCharsets.UTF_8);
 
         return asType(post("/transactions/status")
                 .addHeader("Content-Type", "application/json")
                 .setEntity(body), TypeRef.TRANSACTIONS_STATUSES);
     }
 
-    public List<TransactionStatus> getTransactionsStatuses(Id... ids) throws IOException, NodeException {
-        return getTransactionsStatuses(new ArrayList<>(Arrays.asList(ids)));
+    public List<TransactionStatus> getTransactionsStatuses(Id... txIds) throws IOException, NodeException {
```
Comment (DrBlast) [inline src/main/java/com/wavesplatform/wavesj/Node.java:485]: rename to getTransactionsStatus

File: src/main/java/com/wavesplatform/wavesj/Node.java [hunk 2]
```diff
@@ -540,24 +465,24 @@ public TransactionInfo getTransactionInfo(Id txId) throws IOException, NodeExcep
...
+    public List<TransactionStatus> getTransactionsStatuses(List<Id> txIds) throws IOException, NodeException {
         ObjectNode jsonBody = JSON_MAPPER.createObjectNode();
         ArrayNode jsonIds = jsonBody.putArray("ids");
-        ids.forEach(id -> jsonIds.add(id.toString()));
+        txIds.forEach(id -> jsonIds.add(id.toString()));
         StringEntity body = new StringEntity(JSON_MAPPER.writeValueAsString(jsonBody), StandardCharsets.UTF_8);
 
         return asType(post("/transactions/status")
                 .addHeader("Content-Type", "application/json")
                 .setEntity(body), TypeRef.TRANSACTIONS_STATUSES);
```
Comment (DrBlast) [inline src/main/java/com/wavesplatform/wavesj/Node.java:482]: rename type to TRANSACTIONS_STATUS

**Review by DrBlast — CHANGES_REQUESTED** [review]

File: src/main/java/com/wavesplatform/wavesj/Node.java (hunk 1 above)
Comment (br0x) [reply src/main/java/com/wavesplatform/wavesj/Node.java]: > rename to getTransactionsStatus

Why? Statuses is proper naming for list

**Review by DrBlast — APPROVED** [review]

File: src/main/java/com/wavesplatform/wavesj/Node.java (hunk 1 above)
Comment (msmolyakov) [reply src/main/java/com/wavesplatform/wavesj/Node.java]: > Why? Statuses is proper naming for list

Just for consistency with other functions and name of the endpoint, `/transactions/status`.

**Review by DrBlast — APPROVED** [review]

**Pull request merged**